contracts/claim/Cargo.toml

```diff
@@ -42,6 +42,7 @@ cw-storage-plus = "0.13.2"
 sg-marketplace = { git = "https://github.com/public-awesome/marketplace", version = "0.7.0", features = [
   "library",
 ] }
+cw-controllers = "0.13.2"
```

Comment:
Oh good catch. The reason for `sg-controllers` is to handle our custom `Response` type that is specialized for our custom messages. When I was writing this, the compiler had no problem with this admin controller. I believe its written in a way that handles custom response types. But let me write a test to update the admin to make sure it works.

Reply:
This is because admin controller in CW uses `Response<C>` so C = `StargazeMsgWrapper` and it can handle the custom type. But in cw-controller hooks, `SubMsg` is not generic so it cannot be used. Added a test showing update admin working.